### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by disc3110**
- created vet_clinic db
- created animals table
- added some data to the table
- made some queries

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
+INSERT INTO animals (
+    name
+  , date_of_birth
+  , escape_attempts
+  , neutered
+  , weight_kg
+) VALUES (
+  'Agumon'
+  , 'feb/3/20'
```
Comment (oxenprogrammer) [inline data.sql]: **Changes Required**
- [ ] This date format is not supported on database level. It simple means you cannot perform comparison queries with them. Please let us instead use supported date format.

File: data.sql [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+);
+INSERT INTO animals (
+    name
+  , date_of_birth
+  , escape_attempts
+  , neutered
+  , weight_kg
+) VALUES (
+  'Gabumon'
+  , 'nov/15/18'
```
Comment (oxenprogrammer) [inline data.sql]: Same here

File: data.sql [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+);
+INSERT INTO animals (
+    name
+  , date_of_birth
+  , escape_attempts
+  , neutered
+  , weight_kg
+) VALUES (
+  'Pikachu'
+  , 'jan/7/21'
```
Comment (oxenprogrammer) [inline data.sql]: Same here

File: data.sql [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+);
+INSERT INTO animals (
+    name
+  , date_of_birth
+  , escape_attempts
+  , neutered
+  , weight_kg
+) VALUES (
+  'Devimon'
+  , 'may/12/17'
```
Comment (oxenprogrammer) [inline data.sql]: Same here

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (disc3110) [reply data.sql]: Thanks. I'll change the format. But my queries were actually working with that form
Screen Shot 2021-12-01 at 8 01 30
at

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉✔**
Hello @disc3110 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**